Xamarin.Essentials/Launcher/Launcher.ios.tvos.cs

```diff
@@ -30,47 +31,27 @@ static Task<bool> PlatformTryOpenAsync(Uri uri)
         }
 
 #if __IOS__
+
         static UIDocumentInteractionController documentController;
```

Comment:
Will this leak? Do we get an event when a save is actually made? If not, then maybe add a comment saying this is here to prevent the GC from clearing up but should not be referenced outside the method.

I'll also do some tests to see if we actually need this field is the simple act of having the picker visible prevents the GC. Based on our chats in Discord, it seems it is fine, but I'll still test on my devices.

Reply:
yes, this should be tested. but I haven't found a better option yet

Reply:
I think this is fine for now as we don't have open streams or file handles. it is just strings and uris.